### DISCUSSION TITLE
【TianliGPT】前端不显示，也并未提交数据

### DISCUSSION THREAD

**Discussion by zhiqing0205** [tangly1024/NotionNext#2003] [Q&A] [answered]
**描述bug**
【此项必填】简单说明bug的现象、相关的错误提示、日志等

**复现步骤**
【此项必填】出现这个bug的操作步骤

**期望的正常结果**
【此项必填】希望按这个步骤，正常操作结果是什么

**截图**
【可选】相关的页面，应该的结果

**环境**

- 【必填】NotionNext版本 4.2.3
- 【必填】主题 hexo
- 【必填】部署方案 VPS
- 【可选】操作系统:  macOS
- 【可选】浏览器 chrome

**补充说明**
【可选】与问题相关的其它说明

**Comment by tangly1024** [comment]
解决此问题，需要提供一下网站信息，从而获取日志等其他数据

**Comment by zhiqing0205** [comment]
[博客地址](https://ziuch.com)

**Comment by ChenyqThu** [comment, marked as answer]
tianliGPT目前用不了，需要手动修改TianliGPT.js的代码，有2个问题：
1. 执行：因为Next是静态博客，没有pjax，js中的触发逻辑不对，要修改一下
2. 就算能跑，js中获取博文内容是通过 p 标签筛选的，Next博文页面组织方式不同，无法获取文章，最后只会发送标题和H标签给API总结，总结的内容完全不对。
后来我换了一种方式，依赖第三方不如依赖自己，在博客Database增加 一列 AI Summary，用Notion AI总结或者kimi总结都行，总结完了将内容填入，然后渲染博文的时候同步获取 post.AISummary内容，展示到博客前端，这样可以自己选择总结平台，调整总结文本，速度也比请求API快很多，而且免费。
参考：https://chenge.ink
